packages/venia-ui/lib/components/CartPage/PriceAdjustments/priceAdjustments.js

```diff
@@ -10,6 +10,22 @@ import ShippingMethods from './ShippingMethods';
 
 import defaultClasses from './priceAdjustments.css';
 
+/**
+ * PriceAdjustments is a child component of the CartPage component.
```

Comment:
Since we discussed maintenance issues in the scope of this PR, here's another one. If we refactor `PriceAdjustments` to be generically usable or something, we'd have to go through and update this comment.

I'm cool with documenting the inputs and outputs of these functions but I don't think we should include structure data like this - it's way too easy for it to get out of date.

Reply:
Yea, that is my argument against adding any description at all to these things. The docs should describe the types and the code should describe what it's doing. Any description should be high level at most.